app/elements/account-elements/register-page.html

```diff
@@ -0,0 +1,94 @@
+<link rel="import" href="../../bower_components/paper-input/paper-input.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+<link rel="import" href="account-page.html">
+
+<dom-module id="register-page">
+  <template>
+    <account-page id="account" sub-route="/register" heading="Register" body="[[body]]" on-iron-error="handleRegisterError">
```

Comment:
Shouldn't register have some sort of succes event handler? Resetting to the login screen perhaps?